etl-postgres/src/replication/state.rs

```diff
@@ -360,21 +360,18 @@ pub async fn get_pipeline_table_ids<'c, E>(
 where
     E: PgExecutor<'c>,
 {
-    let rows = sqlx::query!(
+    let ids: Vec<SqlxTableId> = sqlx::query_scalar(
```

Comment:
Drive-by change to a non-compiled query because it's better to just use non-compiled queries outside of `etl-api` for now.